# Perspective primers

You must choose one of the following perspective primers and write a detailed, well-supported perspective on the topic.
Each primer presents a nuanced question with no clear right or wrong answer, encouraging you to explore the literature, form your own opinion, and justify your stance.
Your task is to research your chosen topic thoroughly, present a balanced argument, and provide a well-reasoned perspective supported by current scientific evidence.

## Protein structure prediction

**Primer:** Are *ab initio* protein structure prediction algorithms still relevant in the deep learning era?

*Ab initio* protein structure prediction algorithms can determine the three-dimensional structures of proteins from their amino acid sequences without relying on homologous structures.
These methods often involve intensive computational processes and can be time-consuming.
However, recent advances in deep learning, exemplified by tools like AlphaFold, have dramatically improved the accuracy and efficiency of protein structure predictions, challenging the relevance of traditional *ab initio* approaches.

In the era of deep learning, especially with sophisticated models that leverage vast amounts of data and computational power, protein structure prediction has seen unprecedented advancements.
The question arises: do *ab initio* methods still hold value, or have they been rendered obsolete by these newer, data-driven approaches?
This perspective should touch on the balance between traditional algorithmic approaches and cutting-edge machine learning techniques and their implications for the future of computational structural biology.

Possible discussion points:

-   **Accuracy and Reliability:** Compare the accuracy and reliability of *ab initio* methods with deep learning-based predictions.
 Evaluate situations where one method may outperform the other.
-   **Computational Resources:** Assess the computational demands of *ab initio* methods versus deep learning models, considering accessibility for different research institutions.
-   **Data Dependence:** Discuss the dependence of deep learning models on large datasets and the potential limitations this may impose compared to *ab initio* methods, which do not exclusively rely on prior data.
-   **Innovation and Integration:** Explore how traditional *ab initio* methods can be integrated with deep learning approaches to enhance prediction accuracy and reliability.
-   **Case Studies:** Examine specific case studies where *ab initio* methods have provided unique insights or deep learning models have significantly outperformed traditional approaches.
-   **Future Prospects:** Consider the future of protein structure prediction, including potential advancements in *ab initio* and deep learning methods and their implications for the field.

One can argue for the continued relevance of *ab initio* methods based on their foundational principles, independence from large training datasets, and potential for integration with new technologies.
Conversely, others may emphasize deep learning's transformative impact, highlighting its superior accuracy, efficiency, and the paradigm shift in the field.
<!--
???+ tip "Example papers"
    Here are some scientific articles to help get you started.

    **Primary**

    -   Abramson, J., Adler, J., Dunger, J., Evans, R., Green, T., Pritzel, A., ... & Jumper, J. M. (2024). Accurate structure prediction of biomolecular interactions with AlphaFold3. *Nature*, 1-3. DOI: [10.1038/s41586-024-07487-w](https://doi.org/10.1038/s41586-024-07487-w)
    -   Baek, M., DiMaio, F., Anishchenko, I., Dauparas, J., Ovchinnikov, S., Lee, G. R., ... & Baker, D. (2021). Accurate prediction of protein structures and interactions using a three-track neural network. *Science, 373*(6557), 871-876. DOI: [10.1126/science.abj8754](https://doi.org/10.1126/science.abj8754)
    -   Zhou, X., Zheng, W., Li, Y., Pearce, R., Zhang, C., Bell, E. W., ... & Zhang, Y. (2022). I-TASSER-MTD: a deep-learning-based platform for multi-domain protein structure and function prediction. *Nature Protocols, 17*(10), 2326-2353. DOI: [10.1038/s41596-022-00728-0](https://doi.org/10.1038/s41596-022-00728-0)

    **Opinion**

    -    Outeiral, C., Nissley, D. A., & Deane, C. M. (2022). Current structure predictors are not learning the physics of protein folding. *Bioinformatics, 38*(7), 1881-1887. DOI: [10.1093/bioinformatics/btab881](https://doi.org/10.1093/bioinformatics/btab881)
    -    Kuhlman, B., & Bradley, P. (2019). Advances in protein structure prediction and design. *Nature reviews molecular cell biology, 20*(11), 681-697. DOI: [10.1038/s41580-019-0163-x](https://doi.org/10.1038/s41580-019-0163-x)
    -    Doga, H., Raubenolt, B., Cumbo, F., Joshi, J., DiFilippo, F. P., Qin, J., ... & Shehab, O. (2024). A perspective on protein structure prediction using quantum computers. Journal of Chemical Theory and Computation, 20(9), 3359-3378. DOI: [10.1021/acs.jctc.4c00067](https://doi.org/10.1021/acs.jctc.4c00067)

    **Reviews**

    -   Huang, B., Kong, L., Wang, C., Ju, F., Zhang, Q., Zhu, J., ... & Bu, D. (2023). Protein structure prediction: challenges, advances, and the shift of research paradigms. *Genomics, Proteomics & Bioinformatics, 21*(5), 913-925. DOI: [10.1016/j.gpb.2022.11.014](https://doi.org/10.1016/j.gpb.2022.11.014)
    -   Bertoline, L. M., Lima, A. N., Krieger, J. E., & Teixeira, S. K. (2023). Before and after AlphaFold2: An overview of protein structure prediction. Frontiers in bioinformatics, 3, 1120370. DOI: [10.3389/fbinf.2023.1120370](https://doi.org/10.3389/fbinf.2023.1120370) -->

## Computer-aided drug design

**Primer:** Are molecular dynamics simulations overhyped in drug discovery, or do they provide indispensable insights?

Molecular dynamics (MD) simulations allow researchers to observe the behavior of molecules over time, offering detailed insights into the dynamic nature of protein-ligand interactions.
This technique is often used after initial docking studies to refine and validate the predicted interactions.
However, MD simulations are computationally intensive and require significant expertise to interpret.

MD simulations are typically performed after initial docking studies in the drug development pipeline to validate and refine the predicted protein-ligand interactions.
The question arises: Should researchers invest in computationally expensive and time-consuming MD simulations or proceed directly to wet-lab experiments, which might provide more definitive answers?
This decision point is critical, as it impacts the drug development process's efficiency, accuracy, and cost.

Possible discussion points:

-   **Accuracy and Precision:** Debate the accuracy of MD simulations in predicting real-world molecular interactions compared to static docking models.
-   **Computational Resources:** Consider the computational costs and accessibility of MD simulations for different research institutions.
-   **Predictive Value:** Evaluate how MD simulations can refine docking results and their impact on predicting binding affinities and interaction stability.
-   **Experimental Validation:** Discuss whether the insights gained from MD simulations justify the delay and resources compared to proceeding directly to wet lab experiments after docking.
-   **Case Studies:** Examine specific case studies in which MD simulations have either provided critical insights or been unnecessary in the drug design process.
-   **Future Prospects:** Discuss potential advancements in MD technology and their implications for future drug design, considering both the benefits and limitations.

MD simulations could be indispensable because they can provide detailed dynamic insights and refine docking predictions, enhancing the reliability of subsequent wet lab experiments.
Conversely, others might highlight the practical challenges, such as the computational expense and the potential delays in the drug development timeline, advocating for a more streamlined approach that moves directly from docking to experimental validation.
<!--
???+ tip "Example papers"
    Here are some scientific articles to help get you started.

    **Primary**

    -   Alibay, I., Magarkar, A., Seeliger, D., & Biggin, P. C. (2022). Evaluating the use of absolute binding free energy in the fragment optimisation process. Communications Chemistry, 5(1), 105. DOI: [10.1038/s42004-022-00721-4](https://doi.org/10.1038/s42004-022-00721-4)
    -   Eberhardt, J., Santos-Martins, D., Tillack, A. F., & Forli, S. (2021). AutoDock Vina 1.2.0: New docking methods, expanded force field, and python bindings. Journal of chemical information and modeling, 61(8), 3891-3898. DOI: [10.1021/acs.jcim.1c00203](https://doi.org/10.1021/acs.jcim.1c00203)
    -   Wan, S., Sinclair, R. C., & Coveney, P. V. (2021). Uncertainty quantification in classical molecular dynamics. Philosophical Transactions of the Royal Society A, 379(2197), 20200082. DOI: [10.1098/rsta.2020.0082](https://doi.org/10.1098/rsta.2020.0082)
    -   Sahakyan, H. (2021). Improving virtual screening results with MM/GBSA and MM/PBSA rescoring. Journal of Computer-Aided Molecular Design, 35(6), 731-736. DOI: [10.1007/s10822-021-00389-3](https://doi.org/10.1007/s10822-021-00389-3)
    -   Lee, T. S., Lin, Z., Allen, B. K., Lin, C., Radak, B. K., Tao, Y., ... & York, D. M. (2020). Improved alchemical free energy calculations with optimized smoothstep softcore potentials. Journal of chemical theory and computation, 16(9), 5512-5525. DOI: [10.1021/acs.jctc.0c00237](https://doi.org/10.1021/acs.jctc.0c00237)

    **Opinion**

    -   Song, L. F., & Merz Jr, K. M. (2020). Evolution of alchemical free energy methods in drug discovery. Journal of Chemical Information and Modeling, 60(11), 5308-5318. DOI: [10.1021/acs.jcim.0c00547](https://doi.org/10.1021/acs.jcim.0c00547)

    **Reviews**

    -   Sabe, V. T., Ntombela, T., Jhamba, L. A., Maguire, G. E., Govender, T., Naicker, T., & Kruger, H. G. (2021). Current trends in computer aided drug design and a highlight of drugs discovered via computational techniques: A review. European Journal of Medicinal Chemistry, 224, 113705. DOI: [10.1016/j.ejmech.2021.113705](https://doi.org/10.1016/j.ejmech.2021.113705)
    -   Bassani, D., & Moro, S. (2023). Past, present, and future perspectives on computer-aided drug design methodologies. Molecules, 28(9), 3906. DOI: [10.3390/molecules28093906](https://doi.org/10.3390/molecules28093906)
    -   Yang, C., Chen, E. A., & Zhang, Y. (2022). Protein–ligand docking in the machine-learning era. Molecules, 27(14), 4568. DOI: [10.3390/molecules27144568](https://doi.org/10.3390/molecules27144568)
    -   Dhakal, A., McKay, C., Tanner, J. J., & Cheng, J. (2022). Artificial intelligence in the prediction of protein–ligand interactions: recent advances and future directions. Briefings in Bioinformatics, 23(1), bbab476. DOI: [10.1093/bib/bbab476](https://doi.org/10.1093/bib/bbab476)
    -   Sadybekov, A. V., & Katritch, V. (2023). Computational approaches streamlining drug discovery. Nature, 616(7958), 673-685. DOI: [10.1038/s41586-023-05905-z](https://doi.org/10.1038/s41586-023-05905-z) -->

## Single-cell transcriptomics

**Primer:** In the era of single-cell transcriptomics, are bulk RNA sequencing methods still relevant for understanding gene expression?

Single-cell RNA sequencing (scRNA-seq) has revolutionized our understanding of cellular heterogeneity and gene expression dynamics at unprecedented resolution.
This technology allows researchers to profile the transcriptomes of individual cells, revealing insights into rare cell populations, cellular states, and developmental trajectories previously obscured in bulk RNA sequencing data.

However, bulk RNA sequencing methods, which measure the average gene expression across a population of cells, have been the backbone of transcriptomics research for years.
These methods offer advantages in terms of cost, throughput, and the ability to capture lowly expressed genes.
They also provide a broader view of gene expression patterns across tissues or whole organisms.

As single-cell technologies continue to advance, a critical question emerges: Should researchers invest their resources primarily in single-cell approaches, or do bulk RNA sequencing methods still offer unique and valuable insights that complement single-cell data?

Possible discussion points:

-   **Sensitivity and Depth:** Compare the sensitivity and depth of sequencing between bulk and single-cell methods, considering their ability to detect lowly expressed genes and rare transcripts.
-   **Cost and Accessibility:** Evaluate the cost-effectiveness and accessibility of both approaches for different types of research questions and laboratory settings.
-   **Biological Questions:** Discuss the types of biological questions that are best addressed by bulk RNA-seq versus scRNA-seq and where they might complement each other.
-   **Technical Challenges:** Consider the technical challenges and biases associated with each method, including normalization, batch effects, and data interpretation.
-   **Integrative Approaches:** Explore strategies for integrating bulk and single-cell RNA-seq data to leverage the strengths of both approaches.
-   **Future Developments:** Speculate on how technological advancements might influence the relevance of bulk RNA-seq in the coming years.

Single-cell methods provide such detailed insights into cellular heterogeneity that they supersede the need for bulk approaches in most research contexts.
Conversely, others might emphasize the continued relevance of bulk RNA-seq for specific experimental designs, its ability to capture a more comprehensive picture of gene expression, and its established role in clinical and diagnostic applications.
<!--
???+ tip "Example papers"
    Here are some scientific articles to help get you started:

    **Primary**

    - Zhang, X., Li, T., Liu, F., Chen, Y., Yao, J., Li, Z., ... & Wang, J. (2019). Comparative analysis of droplet-based ultra-high-throughput single-cell RNA-seq systems. *Molecular cell, 73*(1), 130-142. DOI: [10.1016/j.molcel.2018.10.020](https://doi.org/10.1016/j.molcel.2018.10.020)
    - Ding, J., Adiconis, X., Simmons, S. K., Kowalczyk, M. S., Hession, C. C., Marjanovic, N. D., ... & Levin, J. Z. (2020). Systematic comparison of single-cell and single-nucleus RNA-sequencing methods. *Nature biotechnology, 38*(6), 737-746. DOI: [10.1038/s41587-020-0465-8](https://doi.org/10.1038/s41587-020-0465-8)
    - Squair, J. W., Gautier, M., Kathe, C., Anderson, M. A., James, N. D., Hutson, T. H., ... & Courtine, G. (2021). Confronting false discoveries in single-cell differential expression. *Nature communications, 12*(1), 5692. DOI: [10.1038/s41467-021-25960-2](https://doi.org/10.1038/s41467-021-25960-2)

    **Reviews**

    - Stark, R., Grzelak, M., & Hadfield, J. (2019). RNA sequencing: the teenage years. Nature Reviews Genetics, 20(11), 631-656. DOI: [10.1038/s41576-019-0150-2](https://doi.org/10.1038/s41576-019-0150-2)
    - Lähnemann, D., Köster, J., Szczurek, E., McCarthy, D. J., Hicks, S. C., Robinson, M. D., ... & Schönhuth, A. (2020). Eleven grand challenges in single-cell data science. *Genome biology, 21*(1), 31. DOI: [10.1186/s13059-020-1926-6](https://doi.org/10.1186/s13059-020-1926-6)
    - Tritschler, S., Büttner, M., Fischer, D. S., Lange, M., Bergen, V., Lickert, H., & Theis, F. J. (2019). Concepts and limitations for learning developmental trajectories from single cell genomics. *Development, 146*(12), dev170506. DOI: [10.1242/dev.170506](https://doi.org/10.1242/dev.170506) -->
